single-line-diagram-core/src/main/java/com/powsybl/sld/svg/FeederInfo.java

```diff
@@ -44,11 +45,11 @@ public FeederInfo(String componentType, LabelDirection arrowDirection, String le
     }
 
     public FeederInfo(String componentType, double value) {
-        this(componentType, value, null);
+        this(componentType, value, 0, null);
     }
 
-    public FeederInfo(String componentType, double value, String userDefinedId) {
-        this(componentType, value > 0 ? LabelDirection.OUT : LabelDirection.IN, null, String.valueOf(Math.round(value)), userDefinedId);
+    public FeederInfo(String componentType, double value, int precision, String userDefinedId) {
```

Comment:
Could you introduce inside `LayoutParameters` a feeder info precision parameter?